jiant/scripts/preproc/superglue_leaderboard_submission_formatter.py

```diff
@@ -0,0 +1,52 @@
+"""Translate raw prediction files for SuperGLUE tasks into format expected by SuperGLUE leaderboard.
+
+This script helps translate the raw prediction files for SuperGLUE tasks into the jsonl files
+required by the SuperGLUE leaderboard. See https://super.gluebenchmark.com/faq for leaderboard info.
+
+To use this script specify one or more raw prediction input filepath in raw_pred_input_filepaths.
```

Comment:
very minor: To use this script**,** specify...

Reply:
Thanks, done w/ 12e65e4